core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/clazzassignment/detailoverview/ClazzAssignmentDetailOverviewViewModel.kt

```diff
@@ -731,5 +756,7 @@ class ClazzAssignmentDetailOverviewViewModel(
 
         const val DEST_NAME = "CourseAssignmentDetailOverviewView"
 
+        const val SOCIAL_WARNING_WEB_URL = "https://beinternetawesome.withgoogle.com"
```

Comment:
This constant should be declared on a related use case (e.g. ShowSocialWarningUseCase), not in this viewmodel. It should be declared once, such that if this link needs to change, there should be one place to go and change it.

This code as-is does implement single point of truth as per coding style - the same constant is being declared in multiple places.